tests/doctests/wmts.txt

```diff
@@ -14,8 +14,8 @@ Find out what a WMTS has to offer. Service metadata:
     '1.0.0'
     >>> wmts.identification.title
     'NASA Global Imagery Browse Services for EOSDIS'
-    >>> str.strip(wmts.identification.abstract)
-    'Near real time imagery from multiple NASA instruments'
+    >>> str(wmts.identification.abstract.split('.')[0])
```

Comment:
There is a Unicode character in the second part of the string.  I think it is an end single quote character.  I wasn't too sure how to make that work with Python 2 & 3, but I'll try again.